### PR TITLE
Create new HeroSection

### PR DESCRIPTION
**Pull request by grosbouddha**
It is iso-functional with the existing one in kairos.

image

### CODE REVIEW

File: src/layout/section/heroSection/index.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
+import styled from 'styled-components'
+import { responsiveBreakpoints } from '_utils/branding'
+
+import HeroSection from './heroSection'
+
+const StyledHeroSection = styled(HeroSection)`
+  & {
+    background-color: white;
```
Comment (Tuxosaurus) [inline src/layout/section/heroSection/index.tsx]: nit: use branding

File: src/layout/section/heroSection/index.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    display: flex;
+    flex-direction: column;
+  }
+
+  & .hero-info-title.kirk-title {
+    min-height: auto;
+    align-items: flex-end;
+    align-self: center;
+    display: flex;
+    padding-top: 24px;
```
Comment (Tuxosaurus) [inline src/layout/section/heroSection/index.tsx]: nit: use `branding.space`

File: package.json [hunk 3]
```diff
@@ -17,6 +17,7 @@
     "build-cleanup": "rimraf build/src && del \"build/**/story.*\" && del \"build/**/*.unit.*\"",
     "eslint": "eslint \"src/**/*.ts\" \"src/**/*.tsx\"",
     "lint": "git diff --name-only --cached --diff-filter=ACMRTUXB -- \"src/*.ts\" \"src/*.tsx\" | xargs eslint",
+    "lint-fix": "git diff --name-only --cached --diff-filter=ACMRTUXB -- \"src/*.ts\" \"src/*.tsx\" | xargs eslint --fix",
```
Comment (Tip-Sy) [inline package.json]: Oh thanks

File: src/layout/section/heroSection/story.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    heroImageUrl={text(
+      'heroImageUrl',
+      'https://cdn.blablacar.com/kairos/assets/build/images/home_summer_campaign-1ea3207605913c1e26410e605a467eb7.jpg',
+    )}
+    heroImageUrlLarge={text(
+      'heroImageUrlLarge',
+      'https://cdn.blablacar.com/kairos/assets/build/images/home_summer_campaign-1ea3207605913c1e26410e605a467eb7.jpg',
+    )}
+    buttonText={text('buttonText', 'Rechercher un trajet')}
+    buttonHref={text('buttonHref', 'http://google.fr')}
```
Comment (Tip-Sy) [inline src/layout/section/heroSection/story.tsx]: Why not BlaBlaCar website? Oh I get it, you remained Googler in your heart 😜

File: src/layout/section/heroSection/index.tsx (hunk 1 above)
Comment (grosbouddha) [reply src/layout/section/heroSection/index.tsx]: Done

File: src/layout/section/heroSection/index.tsx (hunk 2 above)
Comment (grosbouddha) [reply src/layout/section/heroSection/index.tsx]: Done

**Pull request merged**